metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "puppet-mlocate",
-  "version": "0.0.0-rc1",
+  "version": "0.0.1",
```

Comment:
If this code will be released on the forge, following semver should produce a version `1.0.0`:
https://semver.org/
> Version 1.0.0 defines the public API. The way in which the version number is incremented after this release is dependent on this public API and how it changes.